GRonan_BioC_Ex5.sh

```diff
@@ -0,0 +1,59 @@
+# Solo code for assignment 5 - more for proof of concept and strategizing
```

Comment:
1.	1
2.	1
3.	You present the average wage of those with 12 and 16 years experience instead of the minimum wage. 
a.	0.9


2.9/3